### PR TITLE
Fix broken header validation when receiving a ArtPollReply packet

### PR DESCRIPTION
**Pull request by aklinkert**
TLDR:

This PR fixes a bug which breaks the validation of `ArtPollReply` packages when being in a subnet with the second octet being > 14.

---
Hey Jeroen,

today I stumbled upon a nasty little bug in `go-artnet` during parsing network packages After switching my network setup. The last months (years?) I was working in a subnet of 0.255.0.0/16 due to a limitation in the router I was using, after switching the router I started using the official class a net of 2.0.0.0/8 - and out of a sudden not a single node registered at the controller. I spent several hours of debugging 1) my network setup, 2) my ArtNet nodes and 3) the code and finally found this bug, which get's dropped silently during `Node.recvLoop()`.

So the issue is the `ArtPollReply` being the only packet not having the `ProtVerHi` and `ProtVerLo` and as such the packet can't validate them, so I added an exception for that. Funnily the packet struct itself doesn't have the fields ... anyways, this bug is fixed now, please review and merge this PR.

👋 from Hamburg
Alex

### CODE REVIEW

**Comment by jsimonetti** [comment]
Wow, that is a nasty one!
Thank you for finding it and the fix.
